buckwheat/tokenizer.py

```diff
@@ -333,13 +371,21 @@ class PygmentsParser:
     LEXERS = {"Scala": ScalaLexer(),
               "Swift": SwiftLexer(),
               "Kotlin": KotlinLexer(),
-              "Haskell": HaskellLexer()}
+              "Haskell": HaskellLexer(),
+              "Python": PythonLexer()}
+
     # Pygments token types corresponding to identifiers in a given language.
     IDENTIFIERS = {"Scala": {pygments.token.Name, pygments.token.Keyword.Type},
                    "Swift": {pygments.token.Name},
                    "Kotlin": {pygments.token.Name},
                    "Haskell": {pygments.token.Name, pygments.token.Keyword.Type}}
 
+    # Pygments token types corresponding to comments in a given language.
+    COMMENTS = {"Scala": {pygments.token.Comment},
+                "Kotlin": {pygments.token.Comment},
+                "Haskell": {pygments.token.Comment},
+                "Python": {pygments.token.Comment, pygments.token.Literal.String.Doc}}
```

Comment:
why do we use Pygment for python? Python is supported by tree-sitter

Reply:
because in tree-sitter we cannot extract multi-line comments and multi-line strings which represents comments cannot be extracted either

Reply:
If that's true, this showcases the advantages of having the second parser in the pipeline. TBF, comments do not really really require AST at all, so.

Reply:
> because in tree-sitter we cannot extract multi-line comments and multi-line strings which represents comments cannot be extracted either

There is a way to extract multi-line comments with tree-sitter, but it's kinda non-intuitive, consider this example:

```
>>> multiline_string = '''
... """
... This is multiline comment
... Hmmm yeah
... """
... 
... a = [1, 2, 3]
... '''
>>> cur = parser.parse(multiline_string.encode())
>>> cur.node
<Node kind=expression_statement, start_point=(1, 0), end_point=(4, 3)>
>>> cur.goto_first_child()
True
>>> cur.node
<Node kind=string, start_point=(1, 0), end_point=(4, 3)>
```

So, when there is expression statement with string child at the same position (same `start_point/end_point` or `start_byte/end_byte`, then it's multiline comment.